### PR TITLE
Feature fix accounts tests

### PR DESCRIPTION
**Pull request by bmwachajr**
**What does this PR Do ?**
This PR implements fixes to Accounts tests that need fixing

**Description of Task to be completed?**
The app `hc/accounts` has its unit tests in `hc/accounts/tests.` Some of these tests are incomplete and need fixing and some asserts added.

**What are the relevant GitHub issues/pivotal tracker stories?**
[Issue 1](https://www.pivotaltracker.com/story/show/147010551)

### CODE REVIEW

File: hc/accounts/tests/test_profile.py [hunk 1]
```diff
@@ -18,8 +18,13 @@ def test_it_sends_set_password_link(self):
         self.alice.profile.refresh_from_db()
         token = self.alice.profile.token
         ### Assert that the token is set
-
+        self.assertTrue(token is not None)
         ### Assert that the email was sent and check email content
+        #print(len(mail.outbox))
```
Comment (andela-landia) [inline hc/accounts/tests/test_profile.py]: Remove commented code

File: hc/accounts/tests/test_profile.py [hunk 2]
```diff
@@ -18,8 +18,13 @@ def test_it_sends_set_password_link(self):
...
+        self.assertEqual(len(mail.outbox), 1)
+        email = mail.outbox[0]
+        self.assertTrue("Here's a link to set a password for your account on healthchecks.io:" in email.body)
```
Comment (andela-landia) [inline hc/accounts/tests/test_profile.py]: 👍

File: hc/accounts/tests/test_profile.py (hunk 1 above)
Comment (domiebett) [inline hc/accounts/tests/test_profile.py]: This commented out line is not needed.

File: hc/local_settings_old.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#Database configuration
```
Comment (andela-landia) [inline hc/local_settings_old.py:1]: Add this file to your gitignore as they are your personal settings

**Review by andela-landia — COMMENTED** [review]
Add all local files to your gitignore. and delete them from github using `git rm`

File: hc/local_settings_old.py (hunk 3 above)
Comment (bkiseu) [inline hc/local_settings_old.py:1]: is this file really necessary? Should be in git ignore

File: local_settings.py [hunk 4]
```diff
@@ -0,0 +1,139 @@
+"""
```
Comment (bkiseu) [inline local_settings.py]: this file should be in gitignore

**Comment by andela-landia** [comment]
@bmwachajr Resolve the conflicts above

**Review by andela-landia — CHANGES_REQUESTED** [review]
Remove the local files from github

**Review by andela-landia — APPROVED** [review]

**Pull request merged**